scipy/stats/_hypotests.py

```diff
@@ -1652,18 +1672,18 @@ def tukey_hsd(*args):
     >>> group1 = [28.4, 34.2, 29.5, 32.2, 30.1]
     >>> group2 = [26.1, 28.3, 24.3, 26.2, 27.8]
     >>> result = tukey_hsd(group0, group1, group2)
-    >>> conf = res.confidence_interval()
+    >>> conf = res.confidence_interval(confidence_level=.99)
```

Comment:
I made this change to maintain the relevance of this section. Since the CI is now included in the str output, it makes sense to show how you can calculate alternative confidence intervals with different confidence levels.

In example it also says

with our chosen confidence level

so it makes sense to show where we input the CL we chose.